test/api_integration/apis/index_patterns/runtime_fields_crud/delete_runtime_field/errors.ts

```diff
@@ -56,16 +56,6 @@ export default function ({ getService }: FtrProviderContext) {
       expect(response1.status).to.be(404);
     });
 
-    it('returns error when attempting to delete a field which is not a runtime field', async () => {
```

Comment:
No need of this test as the lookup is in the runtimeFieldMap and not the field list, so there is no way to delete a non-runtime field through API.